tests-e2e/LocalOcw.ts

```diff
@@ -183,7 +183,7 @@ class LocalOCW {
           {
             type:      "redirect",
             match:     /^\/static_shared\//,
-            transform: url => `http://localhost:${env.WEBPACK_PORT}${url}`
+            transform: url => `http://localhost:$3000${url}`
           },
```

Comment:
This should be deleted.

When the e2e tests run, the hugo output is served by a static file server using [`serve-handler`](https://www.npmjs.com/package/serve-handler). The static files have structure:
```
test-sites
|--tmp
   |--dist                                     <-- root directory of "overall" static site
      |--courses
         |--ocw-ci-test-course                 <-- hugo output of one course site 
         |--other-course-does-not-exist-yet    <-- hugo output of another course site
         |--could-add-more 
      |--ocw-ci-test-www                       <-- hugo output of root site, ocw-www
```

Let's say that the tests run on port 3010. Then previously requests to `localhost:3010/static_shared/...` had to be redirected to the webpack output. That's what line 185 was doing. We do not need that anymore.

(We do still need to redirect requests to `localhost:3010/static_shared`, since `static_output` is not at the root of the static site. But that's handled by `OCW_WWW_REWRITE`: **all** requests to `localhost:3010/whatever` are redirected to `localhost:3010/ocw-ci-test-www/whatever`, except when `whatever = courses`.)

Reply:
Yes you're right - I was experimenting with stuff and I had put it back in. I removed it and that resolved the playwright test for dev side.